rosbag2/include/rosbag2/rosbag2.hpp

```diff
@@ -37,19 +37,31 @@ class Rosbag2Node;
 class Rosbag2
 {
 public:
+  /**
+   * Records topics to a bagfile. Subscription happens at startup time, hence the topics must
+   * exist when "record" is called.
+   *
+   * @param file_name Name of the bagfile to write
```

Comment:
I believe we are using the notion of `\param` for doxygen

Reply:
Changed.